.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: ci
+
+on:
+  push:
+    branches: [ master, qa ]
+  pull_request:
+    branches: [ master, qa ]
+
+jobs:
+  test-gomod:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ '1.11.x', '1.12.x', '1.13.x', '1.14.x', '1.15.x']
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install
+      env:
+        GO111MODULE: on
+      run: |
+        go get .
```

Comment:
not sure how much perf matters, but github actions supports caching go modules between builds for decreased build times: https://github.com/actions/cache/blob/main/examples.md#go---modules

Reply:
Nice! Added.